corehq/apps/api/tests.py

```diff
@@ -1700,14 +1699,6 @@ def test_basic(self):
         self.assert_accessible_via_sessions(self.list_endpoint)
 
 
-class ILSLocationResourceTest(APIResourceTest, InternalTestMixin):
-    resource = ILSLocationResource
```

Comment:
does this mean we can remove the `ILSLocationResource`

Reply:
https://github.com/dimagi/commcare-hq/pull/23902/commits/4b5e4562db4cbc9ca55614d123d8c0670e640e38